### PR TITLE
refract(page): getting started page re-organized

### PR DESCRIPTION
**Pull request by deepakgithub11**
background color for blocks is changed to simple light color
border added to each blocks(div)

### CODE REVIEW

File: src/app.scss [hunk 1]
```diff
@@ -85,11 +85,13 @@ div.wpui-column {
 
 /* Docs */
 pre {
-    background-color: rgb(181, 181, 181);
+    background-color: rgb(231 231 231);
     padding: 20px;
+    border: 0.5px solid black;
```
Comment (ajitbohra) [inline src/app.scss:90]: border color should be the shade of grey.

File: src/app.scss [hunk 2]
```diff
@@ -85,11 +85,13 @@ div.wpui-column {
...
+    border-radius: 5px;
 }
 
-.main {
+.getting-started {
```
Comment (ajitbohra) [inline src/app.scss:94]: keep classname generic to allow reuse

File: src/pages/getting-started.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ import gettingStarted from "@wpui/docs/getting-started.md"
 
 function GettingStarted() {
   return (
-    <div className="main">
+    <div className="getting-started">
```
Comment (ajitbohra) [inline src/pages/getting-started.js:14]: keep the classname generic to allow reuse

**Review by ajitbohra — CHANGES_REQUESTED** [review]

**Comment by deepakgithub11** [comment]
new CSS applied for border color

File: src/app.scss [hunk 4]
```diff
@@ -85,8 +85,10 @@ div.wpui-column {
 
 /* Docs */
 pre {
-    background-color: rgb(181, 181, 181);
+    background-color: rgb(231 231 231);
```
Comment (ajitbohra) [inline src/app.scss:88]:
```suggestion
    background-color: #f0f0f0;
```

**Review by ajitbohra — CHANGES_REQUESTED** [review]

**Comment by deepakgithub11** [comment]
background color changed
I request you not to close this branch as the getting started page is not responsive for now
I will continue this branch for responsive commit

**Pull request merged**